makefile

```diff
@@ -425,10 +425,11 @@ ifneq (,$(filter install,$(MAKECMDGOALS)))
 endif
 
 # Source dirs in logical order
-DIRS = general linalg linalg/simd mesh mesh/submesh fem fem/ceed/interface \
-       fem/ceed/integrators/mass fem/ceed/integrators/convection \
-       fem/ceed/integrators/diffusion fem/ceed/integrators/nlconvection \
-       fem/ceed/solvers fem/fe fem/lor fem/qinterp fem/integ fem/tmop
+DIRS = general linalg linalg/simd linalg/batched mesh mesh/submesh fem \
+		 fem/ceed/interface fem/ceed/integrators/mass \
+		 fem/ceed/integrators/convection fem/ceed/integrators/diffusion \
+		 fem/ceed/integrators/nlconvection fem/ceed/solvers fem/fe fem/lor \
+		 fem/qinterp fem/integ fem/tmop
```

Comment:
Try to keep the formatting as before -- there are tabs in the new formatting that should be removed.

Reply:
Reverted in 9d78e8cc2324196b09cbaeb00715224a0341f205.